### PR TITLE
[VCDA-7218] TKG 2.5 templates for clusterctl

### PR DESCRIPTION
**Pull request by sakthisunda**
## Description
TKG 2.5 templates for clusterctl

- 

## Checklist
- [ ] tested locally
- [ ] updated any relevant dependencies
- [ ] updated any relevant documentation or examples

## API Changes
Are there API changes?
- [ ] Yes
- [ X] No

If yes, please fill in the below

1. Updated conversions?
    - [ ] Yes
    - [X ] No
    - [ ] N/A
2. Updated CRDs?
    - [ ] Yes
    - [ ] No
    - [ ] N/A
3. Updated infrastructure-components.yaml?
    - [ ] Yes
    - [ X] No
    - [ ] N/A
4. Updated `./examples/capi-quickstart.yaml`?
    - [ ] Yes
    - [ X] No
    - [ ] N/A
5. Updated necessary files under `./infrastructure-vcd/v1.0.0/`?
  
   - [X] N/A

## Issue
NA
Fixes #
NA

- - -
This change is [Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/606)

### CODE REVIEW

File: templates/cluster-template-v1.26.11-tkgv2.5.0-crs.yaml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+spec:
+  kubeadmConfigSpec:
+    clusterConfiguration:
+      apiServer:
+        certSANs:
+          - localhost
+          - 127.0.0.1
+      dns:
+        imageRepository: projects.registry.vmware.com/tkg # image repository to pull the DNS image from
+        imageTag: v1.9.3_vmware.19 # DNS image tag associated with the TKGm OVA used. This dns version is associated with TKG OVA using Kubernetes version v1.25.7.
```
Comment (arunmk) [inline templates/cluster-template-v1.26.11-tkgv2.5.0-crs.yaml:89]: The description should refer to 1.26.11 for this file, and the appropriate versions for the other files

**Review by arunmk — APPROVED** [review]
LGTM with the change requested in the comment file.

**Review by lzichong — APPROVED** [review]
:lgtm:, agreed with Arun as well where we should ensure the commented versions matches the actual version

> Reviewed 6 of 6 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/606#-:-NsK_3mz-jXt3YqV7okD:b-6ax80l)* status: all files reviewed, 1 unresolved discussion (waiting on @Anirudh9794 and @sakthisunda)

**Review by sakthisunda — COMMENTED** [review]
versions in the comment now match the actuals

> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/606#-:-NsKhL6QDKUCp5taWC9T:bo8k086)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @Anirudh9794 and @lzichong)

**Review by sakthisunda — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/606)* status: 0 of 6 files reviewed, 1 unresolved discussion (waiting on @Anirudh9794, @arunmk, and @lzichong)

___
*[`templates/cluster-template-v1.26.11-tkgv2.5.0-crs.yaml` line 89 at r1](https://reviewable.io/reviews/vmware/cluster-api-provider-cloud-director/606#gh-1515088020:-NsKhUCg5hP1BbVGtIdI:b-896fix) ([raw file](https://github.com/vmware/cluster-api-provider-cloud-director/blob/4272c4c100c7c0930d57c12431ea5277d408453c/templates/cluster-template-v1.26.11-tkgv2.5.0-crs.yaml#L89)):*
<details><summary><i>Previously, arunmk (Arun M. Krishnakumar) wrote…</i></summary><blockquote>

The description should refer to 1.26.11 for this file, and the appropriate versions for the other files
</blockquote></details>

Done.

File: templates/cluster-template-v1.26.11-tkgv2.5.0-crs.yaml (hunk 1 above)
Comment (sakthisunda) [reply templates/cluster-template-v1.26.11-tkgv2.5.0-crs.yaml]: Done.

**Pull request merged**